platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/configurationcache/problems/ConfigurationCacheReport.kt

```diff
@@ -35,9 +38,15 @@ import kotlin.contracts.contract
 @ServiceScope(Scopes.BuildTree::class)
 class ConfigurationCacheReport(
     executorFactory: ExecutorFactory,
-    temporaryFileProvider: TemporaryFileProvider
+    temporaryFileProvider: TemporaryFileProvider,
+    internalOptions: InternalOptions
 ) : Closeable {
 
+    companion object {
+        private
+        val problemHashing = InternalFlag("org.gradle.configuration-cache.internal.report.problem-hashing", false)
```

Comment:
🤔 Perhaps something more explicit and prose-like like `*.report.stacktrace-hashes`?

Reply:
I don't feel strongly about it. You suggestion also sounds good 👍 
Renamed